src/mson.js

```diff
@@ -11,7 +11,7 @@ import { indent } from './filters';
 function getTypeAttributes(element, attributes, parent) {
   let typeAttributes = [];
 
-  if (element.element !== 'string' && !!parent) {
+  if (element.element !== 'string' && (!!parent || element.element === 'array')) {
```

Comment:
Without the `element.element === 'array'` check, `array` wouldn't be added to a type without parent for example in 'can render an array' test case:

```js
it('can render an array', () => {
  const dataStructure = new DataStructure(new ArrayElement(['one', 'two']));
  expect(renderAttributes(dataStructure)).to.equal('+ Attributes (array)\n\n    + one\n    + two\n');
}
```